pkg/async/pool/pool.go

```diff
@@ -273,3 +262,27 @@ type unit struct {
 	Context context.Context
 	Runner  async.Runner
 }
+
+type loggingScheduler struct {
+	Inner Scheduler
+	Name  string
```

Comment:
there is already option for Name, would it make sense to remove it?

Reply:
removing it